apps/back/config/routes.yaml

```diff
@@ -1,3 +1,7 @@
 #index:
 #    path: /
 #    controller: App\Controller\DefaultController::index
+
+app_logout:
+  path: "%app.api.url.prefix%/auth/logout"
+  methods: POST
```

Comment:
Should we always add trailing new line ?

Reply:
we need to install an yaml linter